docs/mxdoc.py

```diff
@@ -468,7 +468,13 @@ def copy_artifacts(app):
 def setup(app):
     # If MXNET_DOCS_BUILD_MXNET is set something different than 1
     # Skip the build step
-    if os.getenv('MXNET_DOCS_BUILD_MXNET') == '1'or _MXNET_DOCS_BUILD_MXNET:
+    env_build_mxnet = os.getenv('MXNET_DOCS_BUILD_MXNET', default=None)
+    should_build_mxnet = _MXNET_DOCS_BUILD_MXNET
+    if env_build_mxnet is not None:
+        print("env MXNET_DOCS_BUILD_MXNET is set to " + str(env_build_mxnet))
+        should_build_mxnet = (env_build_mxnet == '1')
+
+    if should_build_mxnet:
```

Comment:
is this needed?

Reply:
yes, env_variable is overriding the hardcoded value  which is always 1. 
IF env variable is set during runtime that takes priority over hardcoded variable.